lib/cmd_tool_lower.go

```diff
@@ -0,0 +1,79 @@
+package lib
+
+import (
+	"fmt"
+
+	"os"
+
+	"github.com/spf13/pflag"
+)
+
+type CmdToolLowerFlags struct {
+	Help  bool
+	Quiet bool
```

Comment:
why are we using this `Quiet` flag? Is there anything in the output which we want to suppress?

Reply:
We have used Quiet flag to control verbosity basically to suppress error messages, debugging info ..